examples/llama-3/lora-8b.yml

```
@@ -14,7 +14,7 @@ val_set_size: 0.05
 output_dir: ./outputs/lora-out
 
 sequence_len: 4096
-sample_packing: true
+sample_packing: false
```

```suggestion
sample_packing: true
eval_sample_packing: false
```

I think you intended to set eval_sample_packing?

---

You are totally right. Sorry for the mistake